fdbserver/masterserver.actor.cpp

```diff
@@ -1246,6 +1248,24 @@ ACTOR Future<Void> waitForPrev(Reference<MasterData> self, ReportRawCommittedVer
 	return Void();
 }
 
+ACTOR Future<Void> waitForTLogPrev(Reference<MasterData> self, GetTLogPrevCommitVersionRequest req) {
+	// TraceEvent("WaitForTLogPrev").detail("Prev",req.prev).detail("CommitVersion",req.commitVersion).detail("PrevTLogVersion",self->prevTLogVersion.get());
+	if (req.prev) {
+		wait(self->prevTLogVersion.whenAtLeast(req.prev));
+	}
+
+	GetTLogPrevCommitVersionReply reply;
+	for (uint16_t tLog : req.writtenTLogs) {
+		reply.tpcvMap[tLog] = self->tpcvVector[tLog];
+		self->tpcvVector[tLog] = req.commitVersion;
+	}
+	if (req.commitVersion > self->prevTLogVersion.get()) {
```

Comment:
Can this if-condition ever be false?

Reply:
I don't think so after some testing. Removed.